### PR TITLE
Documentation: Elaborate on strict versions of concat, reverse, batchNormalization and pad ops

### PR DESCRIPTION
**Pull request by ManrajGrover**
#### Description

This PR elaborates exposed strict versions of concat, reverse, batchNormalization and pad operations in the documentation.

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1096)

### CODE REVIEW

**Comment by dsmilkov** [comment]
:lgtm_strong: Thanks Manraj!

---

Reviewed 4 of 4 files at r1.
Review status: :shipit: complete! 1 of 1 LGTMs obtained

---



*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1096#-:-LEzhsa3ANqG4x5_gkBW:bxdkrh8)*
